website/docs/routes/university.md

```diff
@@ -1566,6 +1590,44 @@ category 列表：
 
 <Route author="legr4ndk" example="/hdu/cs/pg" path="/hdu/cs/pg" radar="1" rssbud="1"/>
 
+### 教务处 {#hang-zhou-dian-zi-ke-ji-da-xue-jiao-wu-chu}
+
+<Route author="nyaShine" example="/hdu/jwc/tzgg" path="/hdu/jwc/:category?" paramsDesc={['分类']} radar="1">
+
+| 分类 | 通知公告 | 学籍管理 | 考试管理 | 排课选课 | 成绩管理 | 毕业管理 | 实践教学 | 毕业设计 | 实践基地 | 学科竞赛 | 专业建设 | 课程建设 | 教材建设 | 综合管理 | 下载中心 | 教务运行 | 实践教学 | 教学建设 | 其它下载 |
+| -------- | -------- | -------- | -------- | -------- | -------- | -------- | -------- | -------- | -------- | -------- | -------- | -------- | -------- | -------- | -------- | -------- | -------- | -------- | -------- |
+| 分类ID | tzgg | xjgl | ksgl | pkxk | cjgl | bygl | sjjx | bysj | sjjd | xkjs | zyjs | kcjs | jcjs | zhgl | xzzx | jwyx | sjjx_4555 | jxjs_4556 | qtxz |
+
+</Route>
+
+### 研究生院 {#hang-zhou-dian-zi-ke-ji-da-xue-yan-jiu-sheng-yuan}
+
+<Route author="nyaShine" example="/hdu/grs/1764" path="/hdu/grs/:category?" paramsDesc={['分类ID']} radar="1">
+
+| 分类 | 通知公告 | 招生信息 | 招生简章 | 培养信息 | 培养工作规章制度 | 培养方案 | 学位信息 | 学位工作规章制度 | 学位工作国家文件 | 在职硕士招生信息 | 在职硕士招生简章 | 在职硕士规章制度 | 在职硕士课程进修 | 学生工作工作信息 | 学生工作规章制度 | 学生工作学科竞赛 | 就业信息 | 就业工作政策法规 | 就业工作档案查询 | 相关下载 |
+| -------- | -------- | -------- | -------- | -------- | -------- | -------- | -------- | -------- | -------- | -------- | -------- | -------- | -------- | -------- | -------- | -------- | -------- | -------- | -------- | -------- |
+| 分类ID | 1764 | 1721 | 1722 | 1726 | 1727 | 1728 | 1729 | 1730 | 1731 | 1732 | 1733 | 1734 | 1735 | 1736 | 1737 | 1738 | 1739 | 1740 | 1741 | xgxz |
+
+</Route>
+
+## 河北工程大学 {#he-bei-gong-cheng-da-xue}
+
+### 科信学院 {#he-bei-gong-cheng-da-xue-ke-xin-xue-yuan}
+
+<Route author="nyaShine" example="/hebeu/kexin/tpxw" path="/hebeu/kexin/:category?" paramsDesc={['分类ID']} radar="1">
```

Comment:
Please mention the default value of `category` if it's optional.
https://docs.rsshub.app/joinus/new-rss/add-docs